sink/gst_sink_media_manager.cpp

```diff
@@ -19,52 +19,44 @@
  * 02110-1301 USA
  */
 
-#include "sink_media_manager.h"
+#include "gst_sink_media_manager.h"
 
-SinkMediaManager::SinkMediaManager(const std::string& hostname)
+GstSinkMediaManager::GstSinkMediaManager(const std::string& hostname)
```

Comment:
Should be in separate commit.
One commit for interface spit
One commit for renaming